.github/workflows/command-robotpy-pr.yml

```diff
@@ -9,6 +9,10 @@ on:
 
 jobs:
   comment:
+    permissions:
+      issues: write
```

Comment:
Why is `issues` needed?

Reply:
[The API endpoint](https://docs.github.com/en/rest/issues/comments?apiVersion=2022-11-28#create-an-issue-comment) for commenting on pull requests actually only requires write permissions for either `issues` or `pull-requests`.